docs/UserGuide.md

```diff
@@ -225,13 +225,12 @@ You can then access the food item later on for editing, deleting etc.
 
 * Duplicates cannot be added to the food inventory. (Please head to the [glossary](#9-glossary) for an explanation for duplicate food items.)
 * The description and tag can contain a maximum of 50 and 30 characters respectively, including whitespaces.
-* The expiry date must be in the format of `DD-MM-YYYY` or `DD/MM/YYYY`. The year must be between 2020 and 2120, both inclusive.
```

Comment:
Isn't this an important constraint that should be mentioned?

Reply:
Hmm yea but I think the rest said it's ok to omit specific conditions to make it more concise.

Reply:
Alright. Thanks.